engine-tests/src/test/java/org/terasology/metatesting/IntegrationEnvValidationTest.java

```diff
@@ -0,0 +1,59 @@
+package org.terasology.metatesting;
+
+import com.google.common.util.concurrent.Futures;
+import com.google.common.util.concurrent.ListenableFuture;
+import org.junit.jupiter.api.Test;
+import org.terasology.engine.context.Context;
+import org.terasology.engine.entitySystem.entity.EntityManager;
+import org.terasology.engine.integrationenvironment.MainLoop;
+import org.terasology.engine.integrationenvironment.jupiter.IntegrationEnvironment;
+import org.terasology.engine.registry.In;
+
+import static com.google.common.truth.Truth.assertThat;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+
+@IntegrationEnvironment
+public class IntegrationEnvValidationTest {
```

Comment:
```suggestion
public class IntegrationEnvironmentValidationTest {
```
I know that it is a long name already but the abbreviation `env` is unclear here initially. I think it is best to stick to the full name (also will need to rename the file).